src/legacy/core_plugins/data/public/search/tabify/types.ts

```diff
@@ -17,8 +17,16 @@
  * under the License.
  */
 
-import './_get_columns';
-import './_buckets';
-import './_response_writer';
-import './_integration';
-describe('Tabify Agg Response', function() {});
+import { RangeFilterParams } from '../../../../../../plugins/data/public';
+
+/** @internal **/
+export interface TimeRange extends RangeFilterParams {
```

Comment:
TimeRange is the name of a type exported from data plugin. Is there a better name?

Reply:
np, let's rename it.